openmc/statepoint.py

```diff
@@ -451,7 +451,7 @@ def summary(self):
 
     @property
     def with_summary(self):
-        return False if self.summary is None else True
+        return False if self.summary is False else True
```

Comment:
It is used a few times for error checking in `openmc.mgxs`. I use it in a number of places in InferMC which is built atop of `openmc.mgxs`. I can see removing it in place of the `summary` property though.